src/upgrade_step_from_7p2p0.py

```diff
@@ -45,8 +44,66 @@ def perform(self, file_access, logger):
                     file_access.write_file(self.file_name, ["rcptt_*"], "a")
 
             logger.info("Step completed")
-            return self.SUCCESS_CODE
+            return SUCCESS_CODE
+
+        except Exception as e:
+            logger.error("Unable to perform upgrade, caught error: {}".format(e))
+            return ERROR_CODE
+
+
+class UpgradeMotionSetPoints(UpgradeStep):
+    """
+    Changes blocks to point at renamed PVs. Warns about changed setup.
+    """
+
+    def perform(self, file_access, logger):
+        """
+        Perform the upgrade step
+        Args:
+            file_access (FileAccess): file access
+            logger (LocalLogger): logger
+
+        Returns: exit code 0 success; anything else fail
+
+        """
+        try:
+            logger.info("Changing motion set point PVs")
+
+            changer = ChangePVsInXML(file_access, logger)
+            num_of_changes = changer.change_pv_name("COORD1", "COORD0")
+            changer.change_pv_name("COORD2", "COORD1")
+
+            changer.change_pv_name("COORD0:NO_OFFSET", "COORD0:NO_OFF")
+            changer.change_pv_name("COORD1:NO_OFFSET", "COORD1:NO_OFF")
+
+            changer.change_pv_name("COORD0:RBV:OFFSET", "COORD0:RBV:OFF")
+            changer.change_pv_name("COORD1:RBV:OFFSET", "COORD1:RBV:OFF")
+
+            changer.change_pv_name("COORD0:LOOKUP:SET:RBV", "COORD0:SET:RBV")
+            changer.change_pv_name("COORD1:LOOKUP:SET:RBV", "COORD1:SET:RBV")
+
+            if file_access.exists(MOTION_SET_POINTS_FOLDER):
+                print(
+                    "{} folder exists. Motion set point configuration has changed significantly"
+                    " in this version and must be manually fixed".format(MOTION_SET_POINTS_FOLDER)
+                )
+                print(
+                    "See https://github.com/ISISComputingGroup/ibex_developers_manual/"
+                    "wiki/Motion-Set-points#upgrading-from-720 for how to do this"
+                )
+                input("Press any key to confirm this is done.")
+
+            # CoordX:MTR is gone, hard to automatically replace so just raise as issue
+            number_of_motor_refs = changer.change_pv_name("COORD0:MTR", "COORD0:MTR")
```

Comment:
A method should do one thing only. I feel like we should have another method that gets the number of occurrences of a pv in blocks and synoptics.